config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
 
-  get 'static_pages#home'
-  get 'static_pages/help'
-  get 'static_pages/about'
-  get 'static_pages/contact'
-  root 'application#hello'
+  get "users/new"
+  root "static_pages#home"
+  get  "/help",    to: "static_pages#help"
```

Comment:
```suggestion
  get "/help", to: "static_pages#help"
```
tương tự cho nhưng cái khác